app/controllers/users_controller.rb

```diff
@@ -121,6 +121,17 @@ def download_photos
     redirect_to current_user.exported_photos_file.url
   end
 
+  def import_profile
+    if params[:user]==nil
+      flash[:notice] = I18n.t("users.edit.no_file_given")
+      redirect_to edit_user_path
```

Comment:
Style/IdenticalConditionalBranches: Move `redirect_to edit_user_path` out of the conditional.